concepts/observability-strategy/opentelemetry-operator/README.md

```diff
@@ -0,0 +1,89 @@
+Conclusion: either the operator gets used as underlying technology for the kyma abstractions and instrumentation support AND is accessible for custom scenarios for users. Or the operator gets not used. No other option.
+
+## Proposal
+
+- The existing `telemetry-operator` manages an opentelemetry-operator additionally.
+- The current abstraction `TracePipeline` does not reveal into an actual deployment but an `OpenTelemetryCollector` resource configuration. The configuration gets reconciled and with that cannot be influenced by the user. The `opentelemetry-collector` will resolve the configuration into an actual deployment. That part is still optional and the indirection via the opentelemetry-operator is not mandatory. It will be mainly beneficial for the user to copy-over sich configuration in order to realize a custom trace-collector
+- The telemetry-operator reconciles `Instrumentation` resources which can be used by the user instantly for auto-instrumentation and injection of the `trace-collector` reference
+- The user can specify custom `OpenTelemetryCollector` resource for custom exporter handling or any other activities likes running a sidecar.
+- The user can specify custom `Instrumentation` for env injection of custom collectors.
+
+![Architecture](./assets/architecture.drawio.svg)
+
+## POC
+
+### Prerequisites
+- Have a Kyma cluster in version >= 2.10
+- Have a TracePipeline configured with a valid output
+
+
+### Installation
+Install the Otel Operator using helm
+
+```bash
+helm upgrade otel-operator open-telemetry/opentelemetry-operator --version 0.21.1 --install --namespace kyma-system -f otel-operator-values.yaml
+```
+
+Create an Instrumentation configuration with the endpoints to use
+```bash
+kubectl apply -n kyma-system -f instrumentation.yaml
+```
+
+### Instrumentation
+Annotate your app with either:
+```yaml
+annotations:
+  instrumentation.opentelemetry.io/inject-sdk: "kyma-system/kyma" #to inject env variables only
+  instrumentation.opentelemetry.io/inject-nodejs: "kyma-system/kyma" #to inject env variables and nodejs instrumentation library
+  instrumentation.opentelemetry.io/container-names: "myapp,myapp2" #to inject env variables and javaagent
+```
+
+### Findings
+
+The injection of environment variables does not only inject the destination for the exporters but also injects the typical OTEL resource attributes. Also it supports injecting trace propagation and sampling strategies. That is a very convenient feature to configure your workload declarative in case of it is already instrumented with the OTEL SDK.
+
+The auto-instrumentation was tested for java and nodejs. Here, we failed to get the nodejs instrumentation working at all, one application was coming up operational but without instrumentation active. Another one was even not starting up anymore.
```

Comment:
```suggestion
The auto-instrumentation was tested for Java and Node.js. 
- We failed to get the Node.js instrumentation working at all, one application was coming up operational but without instrumentation active. Another one was not even starting up anymore.
```

I'm guessing the Java tests were successful, but it's not written anywhere, so ???